app/components/topic-card/template.hbs

```diff
@@ -4,11 +4,34 @@
   {{/if}}
   {{topic.description}}
 </div>
-<button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'upvote'}}>
-  <div class='icon upvote {{if isUpvoter 'active'}}'></div>
-  <div class='count'>{{topic.upvoteCount}}</div>
-</button>
-<button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'volunteer'}}>
-  <div class='icon volunteer {{if isVolunteer 'active'}}'></div>
-  <div class='count'>{{topic.volunteerCount}}</div>
-</button>
\ No newline at end of file
+
+{{#if topic.talkBy}}
+  <div class="volunteer-container">
+    <img class='avatar' alt="{{topic.talkBy.username}}" title="{{topic.talkBy.username}}" src='{{topic.talkBy.avatar}}' />
+  </div>
+   {{date-format topic.topicDate format="MMMM Do"}}
+   {{#if talkByCurrentUser}}
+     <button class="link-btn" {{action "cancelTalkBy"}}>
+      [{{fa-icon 'times'}}]
+     </button>
+    {{/if}}
+{{else}}
+  <button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'upvote'}}>
+    <div class='icon upvote {{if isUpvoter 'active'}}'></div>
+    <div class='count'>{{topic.upvoteCount}}</div>
+  </button>
```

Comment:
Yea, I agree. I didn't like how it looked with the new stuff. I'll add back and make an issue to re-style the topic card a bit.